test/src/examples_complete_test.go

```diff
@@ -28,7 +28,7 @@ func TestExamplesComplete(t *testing.T) {
 		// and AWS resources do not interfere with each other
 		Vars: map[string]interface{}{
 			"attributes": attributes,
-			"example":    exampleInput,
+			"tfc_agent_token": os.Getenv("TFC_AGENT_TOKEN"),
```

Comment:
@osterman I wanted to get your thoughts on this -- I can't find a similar example so I went this route, although I'm wondering if there's a preferred method. The goal here is to just bring the TFC_AGENT_TOKEN variable in from Github actions basically.

Reply:
Ah yes, that's interesting. I agree with your implementation. However, I realize that others will probably want us to mount an existing secret, which is a more secure pattern. We can wait on implementing that though.